docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -0,0 +1,496 @@
+
+When the local tools manifest file is available, use the following command to automatically download and install those tools locally:
+
+```console
+dotnet tool restore
+```
+
+Run a local tool with the following command:
+
+```console
+dotnet tool run <tool-command-name>
+```
+
+When a local tool is called, dotnet searches for a manifest up the directory structure. When a tool manifest file is found, it is searched for the requested tool. If the tool is found, it includes the information needed to find the tool in the NuGet global packages location. 
+
+If the tool is found in the manifest, but not the cache, the user receives an error. The message will be improved after Preview 1 to request the user run `dotnet tool restore`.
+
+To add local tools to a directory, you need to first create the tool manifest file. After Preview 1, we will offer a mechanism for creating tool manifest files, such as a dotnet new template. For Preview 1 you must create the file named `dotnet-tools.json` with the following contents:
+
+```json
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {}
+}
+```
+
+Once the manifest is created, you can add local tools to it using:
+
+```console
+dotnet tool install <toolPackageId>
+```
+
+This command installs the latest version of the tool, unless another version is specified.  Even if the latest version was automatically chosen, the version of the tool is written into the tool manifest file to allow the correct version of the tool to be restored or run.
+
+The tool manifest file is designed to allow hand editing – which you might do to update the required version for working with the repository.
+
+Here is an example `dotnet-tools.json` file:
+
+```json
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "dotnetsay": {
+      "version": "2.1.4",
+      "commands": [
+        "dotnetsay"
+      ]
+    },
+    "t-rex": {
+      "version": "1.0.103",
+      "commands": [
+        "t-rex"
+      ]
+    }
+  }
+}
+```
+
+To remove a tool from the tool manifest file, run the following command:
+
+```console
+dotnet tool uninstall <toolPackageId>
+```
+
+For both global and local tools, a compatible version of the runtime is required. Many tools currently on NuGet.org target .NET Core Runtime 2.1. To install those globally or locally, you would still need to install the [NET Core 2.1 Runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+
+For more information, see [Local Tools Early Preview Documentation](https://github.com/dotnet/cli/issues/10288).
+
+## Windows desktop
+
+Starting with .NET Core 3.0 Preview 1, you can build Windows desktop applications using WPF and Windows Forms. These frameworks also support using modern controls and Fluent styling from the Windows UI XAML Library (WinUI) via [XAML islands](/windows/uwp/xaml-platform/xaml-host-controls).
```

Comment:
nit: "Fluent" -> "fluent"

Reply:
I was going to do that too but I found it odd it was capitalized in the source material. So I found this reference: https://docs.microsoft.com/en-us/windows/uwp/design/fluent-design-system/